cfgov/templates/welcome.html

```diff
@@ -3,14 +3,42 @@
 
 {% block furniture %}
     <div class="content-wrapper">
+
         <h1>{% trans 'Welcome' %} <br/>
-        You've logged in to the consumerfinance.gov editorial tools.	
+        {% if 'build' in request.site.hostname %}
```

Comment:
Did you try using http://127.0.0.1/login/ ? It works for me. The `request.site` is a Wagtail site object. The site object's hostname is what is being checked against, it doesn't use the hostname from the request's url.